command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -80,6 +103,20 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
+	// automatically set kvDetails to true if a depth or filter
+	// is provided. this allows the user to omit the -kvdetails
+	// flag if they prefer.
+	if c.kvDepth != 0 || c.kvFilter != "" {
```

Comment:
Will it change anything if we set kvDepth to default to two always above?

Reply:
Eh... It gets a little messy in terms of assumptions being made then to the point where we would almost want to remove `-kvdetails` entirely. But honestly, based on how I think this will get used, I would almost rather back out this shortcut behavior entirely are require `-kvdetails` + other `-kv*` flags instead. I think it makes more sense and this will be used infrequently enough that the additional flags being used won't really be cumbersome.

Reply:
I ended up backing this functionality out. It feels a bit too rickety to me. If we decide we really want it in later, it's easy enough to add without changing the advertised public interface at all since this was all unadvertised, implicit behavior.